src/pages/library/Datasets.js

```diff
@@ -149,7 +149,8 @@ const thousandGenomesLowCoverage = () => h(Participant, {
 }, [
   h(ButtonPrimary, {
     href: Nav.getLink('data-explorer-public', { dataset: '1000 Genomes' }),
-    tooltip: browseTooltip
+    tooltip: browseTooltip,
+    'aria-label': 'Browse 1000 Genomes Low Coverage'
```

Comment:
TOL: I think this is a case where we could do a better job of annotating the page contents, and doing that would make it more accessible in addition to enabling the integration test. Conceptually, this page contains a list of things. If we marked that up with the `list`/`listitem` roles, we could write the selector as "within the listitem containing '1000 Genomes Low Coverage', click 'Browse data'". That's how a real user will ultimately navigate this page, so the more we can mirror that the better.

Reply:
I'd be happy to do this in another ticket - it feels a bit out of scope for this one. Do you have any issues with me creating a separate ticket for this?

Reply:
Yep, separately is fine, I just wanted to bring up the idea.